lab-christian/model/gallery.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const gallerySchema = Schema({
+  name: { type: String, required: true },
+  desc: { type: String, required: true },
+  timestamp: { type: Date, default: Date.now },
```

Comment:
Why does writing ` Date.now` work here? It absolutely does, but what is happening behind the scenes?